### PR TITLE
[fix] Dockerfile build `apt update` failing when running in Docker < 20.10.9

### PR DESCRIPTION
**Pull request by phoinixgrr**
#### Summary

This is a Workaround for Ubuntu 22.04 `apt update` failing when building under Docker < 20.10.9 

TLDR: Ubuntu 22.04 uses an unsupported SystemCall named `clone3` by Glibc >= 2.34, when running `apt-get update` / `apt update`

This causes the build to fail with:
```bash
E: Problem executing scripts APT::Update::Post-Invoke 'rm -f /var/cache/apt/archives/*.deb /var/cache/apt/archives/partial/*.deb /var/cache/apt/*.bin || true'
E: Sub-process returned an error code
The command '/bin/bash -c set -ex &&     apt-get update &&     apt-get install --no-install-recommends -y ffmpeg=7:4.4.2-0ubuntu0.22.04.1 pulseaudio=1:15.99.1+dfsg1-1ubuntu2       xvfb=2:21.1.3-2ubuntu2.3 wget=1.21.2-2ubuntu1 unzip=6.0-26ubuntu3.1 fonts-emojione=2.2.6+16.10.20160804-0ubuntu2 ca-certificates=20211016 &&     wget --progress=dot:giga https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb &&     apt-get update &&     apt-get install --no-install-recommends -y ./google-chrome-stable_current_amd64.deb &&     wget --progress=dot:giga -N http://chromedriver.storage.googleapis.com/2.46/chromedriver_linux64.zip &&     unzip chromedriver_linux64.zip &&     chmod +x chromedriver &&     mv -f chromedriver /usr/local/bin/chromedriver &&     rm chromedriver_linux64.zip &&     rm google-chrome-stable_current_amd64.deb &&     apt-get clean &&     rm -rf /var/lib/apt/lists/* &&     adduser root pulse-access &&     mkdir -pv ~/.cache/xdgr' returned a non-zero code: 100
07:14:10 [FAIL]
make: *** [Makefile:145: docker-build] Error 1
```
https://git.internal.mattermost.com/mattermost/ci-only/calls-recorder/-/jobs/1215239

Fixed in Docker >= 20.10.9 
Since the internal Build system running Docker < 20.10.9  is affected, we opt to introduce this workaround as a low hanging fruit, before updating the build system dockerd to Docker >= 20.10.9 which contains the fix. 

Reference: 
https://stackoverflow.com/questions/71941032/why-i-cannot-run-apt-update-inside-a-fresh-ubuntu22-04

#### Ticket Link

**Ticket:** https://mattermost.atlassian.net/browse/CLD-4589

### CODE REVIEW

**Review by stylianosrigas — APPROVED** [review]

**Review by spirosoik — APPROVED** [review]

**Review by stafot — APPROVED** [review]

**Review by streamer45 — APPROVED** [review]
Thanks! :+1:
